examples/devdata_cli/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/clarify/clarify-go/devdata_cli
+
+go 1.19
+
+require (
+	github.com/clarify/clarify-go v0.2.0
+	github.com/peterbourgon/ff/v3 v3.3.0
+	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
+)
+
+require (
+	github.com/golang/protobuf v1.5.2 // indirect
+	golang.org/x/net v0.0.0-20220621193019-9d032be2e588 // indirect
+	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb // indirect
+	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
+)
+
+replace github.com/clarify/clarify-go v0.2.0 => ../../
```

Comment:
Require some changes, so maybe want to bump this to `v0.2.1` after merge of #12.